docs/source/release_process.rst

```diff
@@ -0,0 +1,45 @@
+Creating a new Release of the SDK
```

Comment:
Just a minor point: would this be more clear about which SDK - `Creating a new release of the Data Attribute Recommendation Python SDK`?

Reply:
Also, I do not feel release needs to be capitalised, but this is just my opinion.